FinancialTransactionsApi/V1/UseCase/ExportSelectedItemUseCase.cs

```diff
@@ -26,7 +26,7 @@ public async Task<byte[]> ExecuteAsync(TransactionExportRequest request)
             {
                 foreach (var item in request.SelectedItems)
                 {
-                    var rId = await _gateway.GetTransactionByIdAsync(request.TargetId, item).ConfigureAwait(false);
+                    var rId = await _gateway.GetTransactionByIdAsync(item).ConfigureAwait(false);
```

Comment:
Why has this UseCase file needed to be changed for this PR?

Reply:
Because we remove the TargetId prop from GetTransactionByIdAsync, and this useCase it's calling this endpoint then we are getting an error.